content.sol

```diff
@@ -117,13 +120,20 @@ contract Content is Ownable {
     //the status is logged in an event at the end of the accessComplete function
     // behavior is currently unchanged regardless of result.
     // 0 indicates that the finalization can proceed.
+    // 100 indicates that the default logic for finalization should be used
     // Other numbers can be used as error codes and would stop the processing.
     function runFinalize(uint256 /*request_ID*/, uint256 /*score_pct*/) public payable returns (uint) {
-        return 0;
+        return 100;
     }
 
     function runFinalizeExt(uint256 requestID, uint256 score_pct, address originator) public payable returns (uint) {
-        return 0;
+        return 100;
     }
 
+    function commandKill() public  {
```

Comment:
We need to protect against another arbitrary content object setting its custom contract address to our custom contract and then killing it